src/styles/card/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+  }
+
+  &-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+  }
+
+  &-container__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    margin: 2px 0 0;
+    color: #060b35;
+  }
+
+  &--0 {
```

Comment:
The class `stars--0` is a modifier, but it is not clear what it modifies. Consider renaming it to clarify its purpose. Also, remember that according to BEM, a modifier should not be used standalone without the block or element it modifies.